src/Dockerfile

```
@@ -1,64 +1,74 @@
+    HELM2_URL="${HELM_RELEASES_URL}/helm-${HELM2_VERSION}-linux-amd64.tar.gz" && \
+    HELM3_URL="${HELM_RELEASES_URL}/helm-${HELM3_VERSION}-linux-amd64.tar.gz" && \
     KUSTOMIZE_URL="${KUSTOMIZE_RELEASES_URL}/kustomize%2F${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz" && \
-    IMAGE_CLEANUP_URL="${IMAGE_CLEANUP_RELEASES_URL}/${IMAGE_CLEANUP_VERSION}/image-cleanup_linux_amd64" && \
+    SEISO_URL="${SEISO_RELEASES_URL}/${SEISO_VERSION}/seiso_linux_amd64" && \
     KUBEVAL_URL="${KUBEVAL_RELEASES_URL}/${KUBEVAL_VERSION}/kubeval-linux-amd64.tar.gz" && \
     SOPS_URL="${SOPS_RELEASES_URL}/${SOPS_VERSION}/sops-${SOPS_VERSION}.linux" && \
     cd /tmp && \
     curl -sSL "$URL" -o /tmp/oc.tgz && \
-    curl -sSL "$HELM_URL" -o /tmp/helm.tgz && \
+    curl -sSL "$HELM2_URL" -o /tmp/helm2.tgz && \
+    curl -sSL "$HELM3_URL" -o /tmp/helm3.tgz && \
     curl -sSL "$KUSTOMIZE_URL" -o /tmp/kustomize.tgz && \
-    curl -sSL "$IMAGE_CLEANUP_URL" -o /tmp/image-cleanup && \
+    curl -sSL "$SEISO_URL" -o /tmp/seiso && \
     curl -sSL "$KUBEVAL_URL" -o /tmp/kubeval.tgz && \
     curl -sSL "$JQ_URL" -o /tmp/jq && \
     curl -sSL "$SOPS_URL" -o /bin/sops && \
     echo "${SHA256SUM}  /tmp/oc.tgz" > /tmp/CHECKSUM && \
-    echo "${HELM_SHA256SUM}  /tmp/helm.tgz" > /tmp/HELM_CHECKSUM && \
+    echo "${HELM2_SHA256SUM}  /tmp/helm2.tgz" > /tmp/HELM2_CHECKSUM && \
+    echo "${HELM3_SHA256SUM}  /tmp/helm3.tgz" > /tmp/HELM3_CHECKSUM && \
     echo "${KUSTOMIZE_SHA256SUM}  /tmp/kustomize.tgz" > /tmp/KUSTOMIZE_CHECKSUM && \
-    echo "${IMAGE_CLEANUP_SHA256SUM}  /tmp/image-cleanup" > /tmp/IMAGE_CLEANUP_CHECKSUM && \
+    echo "${SEISO_SHA256SUM}  /tmp/seiso" > /tmp/SEISO_CHECKSUM && \
     echo "${KUBEVAL_SHA256SUM}  /tmp/kubeval.tgz" > /tmp/KUBEVAL_CHECKSUM && \
     sha256sum -c /tmp/CHECKSUM && \
-    sha256sum -c /tmp/HELM_CHECKSUM && \
+    sha256sum -c /tmp/HELM2_CHECKSUM && \
+    sha256sum -c /tmp/HELM3_CHECKSUM && \
     sha256sum -c /tmp/KUSTOMIZE_CHECKSUM && \
-    sha256sum -c /tmp/IMAGE_CLEANUP_CHECKSUM && \
+    sha256sum -c /tmp/SEISO_CHECKSUM && \
     sha256sum -c /tmp/KUBEVAL_CHECKSUM && \
     tar -xzvf /tmp/oc.tgz && \
-    tar -xzvf /tmp/helm.tgz && \
+    tar -xzvf /tmp/helm2.tgz && \
+    mv -v "/tmp/linux-amd64/helm" /bin/helm2 && \
+    rm -rf -v "/tmp/linux-amd64" && \
+    tar -xzvf /tmp/helm3.tgz && \
+    mv -v "/tmp/linux-amd64/helm" /bin/helm3 && \
     tar -xzvf /tmp/kustomize.tgz && \
     tar -xzvf /tmp/kubeval.tgz && \
-    chmod 755 /tmp/kustomize /tmp/image-cleanup /tmp/jq /bin/sops && \
+    chmod 755 /tmp/kustomize /tmp/seiso /tmp/jq /bin/sops && \
     mv -v "/tmp/${ARCHIVE}/oc" /bin/ && \
-    mv -v "/tmp/linux-amd64/helm" /bin/ && \
     mv -v "/tmp/kubeval" /bin/ && \
     mv -v "/tmp/kustomize" /bin/ && \
-    mv -v "/tmp/image-cleanup" /bin/ && \
+    mv -v "/tmp/seiso" /bin/ && \
     mv -v "/tmp/jq" /bin/ && \
     ln -s /bin/oc /bin/kubectl && \
+    ln -s /bin/helm2 /bin/helm && \
```

I spent some time in the shower :shower: this morning doing some heaving thinking. :thinking: I figured that, as we do _two_ upgrades in the PR, we need _two_ deprecations to be added! :bulb: (In fact, that's exactly what we wrote down in the acceptance criteria in VINT-1816. :grimacing:)

Hence, this symlink should point to a script, say `/opt/helm2-deprecation.sh` that spits out a loud deprecation warning :loudspeaker: before calling `/bin/helm2` with all parameters it got passed to.

 ```suggestion
    ln -s /opt/helm2-deprecation.sh /bin/helm && \
```
I'll try to add a commit with such a script, if you don't mind, so we can move on.

---

Added the change with a new commit.